var/spack/repos/builtin/packages/r-dupradar/package.py

```diff
@@ -6,17 +6,18 @@
 from spack.package import *
 
 
-class BioconductorDupradar(RPackage):
+class RDupradar(RPackage):
     """Assessment of duplication rates in RNA-Seq datasets"""
 
     homepage = "https://bioconductor.org/packages/3.16/bioc/html/dupRadar.html"
     url = "https://bioconductor.org/packages/release/bioc/src/contrib/dupRadar_1.30.0.tar.gz"
     maintainers("pabloaledo")
 
-    bioc = "dupradar"
+    bioc = "dupRadar"
 
     version("1.30.0", sha256="a299d7a4578047dfc19237e34255b0f50f70ce41d29762ef9f5a7741ba35aa3d")
```

Comment:
For all 4 recipes, please update the pre-existing version, so that it uses the corresponding `commit` ID and not the `sha256`.